src/transformStateWithClones.js

```diff
@@ -7,7 +7,32 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  let newState = { ...state };
```

Comment:
[NAMING]: The variable `newState` is not a good choice, it is better to use a more descriptive name. For example, `stateCopy` would be a better choice because it clearly indicates that this is a copy of the original state.